src/css/style.scss

```
@@ -17,6 +17,11 @@
   h2 {
```

ich adde an der stelle immer gerne noch eine klasse `.h2`

also im code würde es so aussehen. 
Faktisch ist h[] ja ein html element und beschreibt nicht umbedingt das styling. das h[] styling wiederum kann auch von anderen html elementen benutzt werden und wiederverwendet werden. so greift man dem usecase schon mal vor

```suggestion
  h2,
  .h2 { ... }
```